qamomile/core/circuit/circuit.py

```diff
@@ -134,6 +135,16 @@ class ThreeQubitGate(Gate):
     control2: int
     target: int
 
+@dataclasses.dataclass
+class ParametricExpGate(Gate):
+    r"""Parametric exponential gate class.
+    .. math::
+            e^{-i\theta H}
+    """
```

Comment:
It it better to explain what $\theta$ and $H$ mean.
Because we only accept $H$ is the Hamiltonian case.